frontend/src/helpers/object-to-query.ts

```diff
@@ -1,16 +1,52 @@
 export const objectToQueryString = (filters: {
-  [p: string]: string | string[];
+  [p: string]: string | string[] | Record<string, string | string[]>[];
 }): string[][] => {
   const notEmpties = Object.fromEntries(
     Object.entries(filters).filter(
-      ([_, value]) => value.length >= 1 && value !== '' && value[0] !== '',
+      ([_, value]) =>
+        value.length >= 1 &&
+        value !== '' &&
+        Array.isArray(value) &&
+        value[0] !== '',
```

Comment:
> `value.length >= 1 && value !== ''`

:eye::eye:

Also, you don't need to make a comparison with the `''`. You can just check `!value` or `!value[0]`